spec/policies/v2/tailoring_policy_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe V2::TailoringPolicy do
+  let(:canonical_profile) do
+    FactoryBot.create(
+      :v2_profile,
+      os_major_version: 9,
+      ref_id_suffix: 'foo',
+      supports_minors: [8]
+    )
+  end
+  let(:user) { FactoryBot.create(:v2_user) }
+  let!(:item) do
+    FactoryBot.create(
+      :v2_tailoring,
+      policy: FactoryBot.create(
+        :v2_policy,
+        account: user.account,
+        profile: canonical_profile
+      ),
+      profile: canonical_profile,
+      os_minor_version: 8
+    )
+  end
+
+  before do
+    FactoryBot.create(
+      :v2_tailoring,
+      policy: FactoryBot.create(
+        :v2_policy,
+        account: FactoryBot.create(:v2_user).account,
+        profile: canonical_profile
+      ),
+      profile: canonical_profile,
+      os_minor_version: 8
+    )
+  end
```

Comment:
Why is this here, what's its purpose?

Reply:
The tailoring is used in the test that a user can't access foreign tailoring. 
Notice the account is changed (not `user.account`) in this instance.